gr-uhd/lib/CMakeLists.txt

```diff
@@ -20,13 +20,29 @@
 ########################################################################
 # Setup library
 ########################################################################
-add_library(gnuradio-uhd
+set(LIB_GR_UHD_SOURCES
     usrp_block_impl.cc
     usrp_source_impl.cc
```

Comment:
since you're reordering headers into alphabetical order, would you please also reorder these?